pulsar-client-cpp/include/pulsar/c/client_configuration.h

```diff
@@ -25,7 +25,13 @@
 extern "C" {
 #endif
 
-typedef enum { pulsar_DEBUG = 0, pulsar_INFO = 1, pulsar_WARN = 2, pulsar_ERROR = 3 } pulsar_logger_level_t;
+typedef enum
+{
+    pulsar_DEBUG = 0,
+    pulsar_INFO = 1,
+    pulsar_WARN = 2,
+    pulsar_ERROR = 3
+} pulsar_logger_level_t;
```

Comment:
Are these changes necessary? Don't format your code if you don't have to

Reply:
This change might be caused by the clang-format version. Currently Pulsar C++ client uses clang-format 5.0, whose behavior is a little different from higher versions of clang-format.

Please reformat your code with clang-format 5.0. You can simply run `docker-format.sh`.

Reply:
@billowqiu PTAL.

Reply:
> 

I'll start with a Linux environment，i see the ci os is  Ubuntu  20.04.3